app/templates/image-set/cv-results.hbs

```diff
@@ -0,0 +1,44 @@
+<h3>CV Results</h3>
+
+<div class='row'>
+  <div class='col-sm-4'>
+    <ul class='list-group'>
+      {{#each model as |cvResult|}}
+        {{lg-cv-result-summary cvResult=cvResult action='setActiveCvResult'}}
+      {{/each}}
+    </ul>
+  </div>
+
+  <div class='col-sm-4'>
+    <div class='row'>
+      {{#unless searchImageSet.lion}}
+        {{#if showLionNameField}}
+          {{input type='text' name='lionName' placeholder='Lion Name' value=lionName}}
+          <button class='btn btn-xs btn-warning create-lion' {{action 'createLion'}}>
+            Create Lion
```

Comment:
I think as we work through the UI we'll bring the buttons in line so the expectation is more in match with the user experience. In the end we don't want to have a red button that means delete as well as a red button that means create, or a green that means delete. I'll make a note to check on these.